Liminal Express Developer_1.3.1 (v1).postman_collection.json

freyasaha-ship-it opened the pull request:
Liminal Express Developer_1.3.1 (v1)

freyasaha-ship-it committed:
Liminal Express Developer_1.3.1 (v1)

Liminal Express Developer_1.3.1 (v1)

rezaliminal commented on Liminal Express Developer_1.3.1 (v1):
```diff
@@ -0,0 +1,2460 @@
+		"name": "Liminal Express Developer_1.3.1 (v1)",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json",
+		"_exporter_id": "48100634",
+		"_collection_link": "https://lmnl88.postman.co/workspace/Lmnl-Workspace~57121a3f-e0bb-48fb-88d4-908a5c3146bc/collection/36828385-dc7722a2-3a01-4b3e-8d7d-9aa8137dae60?action=share&source=collection_link&creator=35100055"
```
@freyasaha-ship-it please remove this collection link, because this is used for internal only

freyasaha-ship-it replied:
Removed info.collection variable

freyasaha-ship-it committed:
Update Liminal Express Developer_1.3.1 (v1)

Removed info.collection variable as per review feedback since it is internal-only.
